retriever/lib/datasets.py

```diff
@@ -28,3 +35,12 @@ def datasets(arg_script=None):
     all_scripts = sorted(all_scripts, key=lambda s: s.lower())
 
     return all_scripts
+
+def license(dataset):
```

Comment:
scripts = script_dict()
This could be a function that test if a script is available.
It will be used by both `ls` and `citation` and `license`.
The function could take a string and convert it to lowercase.
An in the function we could handle the errors.


```
    if dataset in scripts:
        script = scripts[dataset]
    else:
        raise UnknownDataset(dataset)
```